spec/components/chart_spec.rb

```diff
@@ -8,7 +8,8 @@
       table_label: "Page views table",
       chart_id: 'pviews_jan_chart',
       table_id: 'pviews_jan_table',
-      keys: %w(Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec),
+      #keys: %w(Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec),
```

Comment:
May as well delete this line.

Reply:
fixed
(sorry, I'm getting annoyed with myself for constantly missing things I should have removed)

Reply:
No problem :)